README.md

```diff
@@ -96,6 +96,15 @@ $ git config --get user.email
 $ smimesign --list-keys
 ```
 
+**Add smimesign options**
+
+Currently only `tsa` and `include-certs` options are supported.
+
+```bash
+$ git config --global gpg.x509.smimesign.tsa http://timestamp.digicert.com
+$ git config --global gpg.x509.smimesign.include-certs -1
```

Comment:
The docs should probably note what the values are for `include-certs` and what the recommended value should be.

Reply:
@clairernovotny Thank you for the feedback! Fully agree this is missing here and also within the `README.md`. The options are only within the source code: https://github.com/github/smimesign/blob/master/main.go#L31 . I do add that information.